### DISCUSSION TITLE
Contact sheet outputting too small

### DISCUSSION THREAD

**Discussion by treeriot** [ImageMagick/ImageMagick#2369] [Help] [answered]
Why is my contact sheet outputting to roughly 4x5in when I've listed in the code for it to be 8.5x11in (2550x3300 pixels) at 300 dpi?

![Screen Shot 2020-08-04 at 5 01 23 PM](https://user-images.githubusercontent.com/69062957/89344699-46c3aa00-d674-11ea-885d-e4247dbe995f.png)

**Comment by fmw42** [comment, marked as answer]
-page is a setting and does not resize the image. If you want to resize it, then use -resize.  You have resized each image to 200x200 and tiled 6x7. So that is approx a size of 1200x1400. Divide by 300 dpi = 4x4.67 in. Remove the -page as it does nothing. Change the 200x200 size in -geometry to 425x471  (2550/6=424 and 3300/7=471.4). Or simply resize the montage.

**Reply by treeriot** [reply]
Thank you!!!
